src/MovieDetails/MoviePage.js

```diff
@@ -31,29 +20,21 @@ class MoviePage extends Component {
     try {
       const movie = await getSingleMovieDetails(this.props.foundMovieId)
       const movieInfo = movie.movie
-
+      console.log(movieInfo)
       this.setState({
-        average_rating: movieInfo.average_rating,
-        backdrop_path: movieInfo.backdrop_path,
-        budget: movieInfo.budget,
-        genres: movieInfo.genres,
-        id: movieInfo.id,
-        overview: movieInfo.overview,
-        poster_path: movieInfo.poster_path,
-        release_date: movieInfo.release_date,
-        revenue: movieInfo.revenue,
-        runtime: movieInfo.runtime,
-        tagline: movieInfo.tagline,
-        title: movieInfo.title,
+        movie: movieInfo
       })
     } catch (error) {
       this.setState({error: error})
     }
+    console.log(this.state.movie)
+    console.log(this.state)
     this.findUserRatingsForFoundMovie()
   }
 
   formatBudgetAndRevenue(x) {
-    if (x === 0) {
+    console.log(x)
+    if (x === 0 || x === undefined) {
```

Comment:
I think we can refactor this to say `(!x)`, which would encompass both of these conditions!

Reply:
great catch!